tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "types": ["node", "jest", "symbol-observable"],
```

Comment:
addresses issues with running `yarn test`, I'm happy to remove these if the issue is specific to my environment.